styleguide/core/article/ListItem.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I think this example should be hidden from the styleguide navigation. Would you prefix the filename with an `_`?